org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/mock/MockMultipleClientTest.java

```diff
@@ -19,6 +19,9 @@
 import org.junit.Before;
 import org.junit.Ignore;
 
+/**
+ * TODO DAQ-1004 see comment in superclass
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)